src/main/java/no/bibsys/lambda/deploy/handlers/SwaggerHubUpdater.java

```diff
@@ -0,0 +1,112 @@
+package no.bibsys.lambda.deploy.handlers;
+
+import com.amazonaws.services.apigateway.AmazonApiGateway;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Optional;
+import no.bibsys.apigateway.ApiGatewayApiInfo;
+import no.bibsys.cloudformation.CloudFormationConfigurable;
+import no.bibsys.cloudformation.Stage;
+import no.bibsys.swaggerhub.ApiDocumentationInfo;
+import no.bibsys.swaggerhub.SwaggerDriver;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+
+public class SwaggerHubUpdater {
```

Comment:
Maybe a short doc about purpose and expected usage for this class?

Reply:
done